### ISSUE TITLE
--webroot run command not creating authenticating directory/file (windows)

### ISSUE THREAD

**Issue by ifny** [xenolf/lego#504]
Version:
`lego version v0.4.1-0-g67c86d860a79`

Command in CMD:
`lego -m="example@foo.bar" --csr=./SSL/MyCSR.pem --http=":5000" --tls=":5001" --webroot="./www_root/" run `

Response:
```
2018/03/19 16:44:46 [INFO][foo.bar] acme: Obtaining bundled SAN certificate given a CSR
2018/03/19 16:44:47 [INFO][foo.bar] AuthURL: https://acme-v01.api.letsencrypt.org/acme/authz/V2DjRd4m9-zIyl1xRZK9LdQItvx30SUqQGsKHOafOhI
2018/03/19 16:44:47 [INFO][foo.bar] acme: Trying to solve HTTP-01
2018/03/19 16:44:49 [foo.bar] Could not obtain certificates
        acme: Error 403 - urn:acme:error:unauthorized - Invalid response from http://foo.bar/.well-known/acme-challenge/fvUTkgl9m7DPIQq7yaHbIne9aCNfOSLsLLtJze8h3y4 [672.223.151.155]: 404
Error Detail:
        Validation for foo.bar:80
        Resolved to:
                672.223.151.155
        Used: 672.223.151.155
```
There is also no directories created called _.well-known/acme-challenge_. If I create this directory, no file is created.

**Comment by seonixx** [comment]
Did you ever resolve this?

**Comment by ifny** [comment]
I'm afraid not, which led to me using a different LE client.

**Comment by ldez** [comment]
Fixed by #633
